ext/hal/nordic/nrfx_config.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2017, Nordic Semiconductor ASA
+ * All rights reserved.
+ * 
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
```

Comment:
Should this file have been added in the import commit?

Reply:
I don't think do. This file was not imported, it was created specially for Zephyr. nrfx provides only templates of configuration files.